public/css/login.css

```diff
@@ -1,3 +1,85 @@
-h1{
-  color: red;
+*,html{
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+body{
+  background: #dddddd3b;
+}
+.left {
+  width: 50%;
+  float: right;
+  padding: 0;
+  margin: 0;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.loginForm {
+  position: absolute;
+    width: 300px;
+    height: 450px;
+    top: 50%;
+    left: 50%;
+    transform: translate(50%,-50%);
+    box-sizing: border-box
+}
+.login {
+  width: 100px;
+  height: 100px;
+  margin: 0 auto;
+}
+.login i {
```

Comment:
@AhmedAl-Almi I would suggest using the class name for this selector (for example `.login .fas.fa-user {` instead of the `i`.  I think it'd be better to reserve the `i` tag just for making text italic, since that's how devs typically expect use it.